README.md

```diff
@@ -26,23 +31,23 @@ A list of packages to install with pip. Examples below:
         version: "1.2.3"
       - name: awscli
         version: "1.11.91"
-    
+
```

Comment:
Please leave these spaces in; even though trailingspaces would highlight these spaces, they are part of the code indentation for the README block of code.

Reply:
Oh god, I must find a text editor, that will not clear it automatically for me :) Will do now.